app/views/page/_two_to_one_image_component.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="grid-row grid-gap display-flex flex-column flex-row-tablet">
+  <% if component.text_alignment == "Left" %>
+    <div class="<%= component.flipped_ratio? ? 'tablet:grid-col-8' : 'tablet:grid-col-4' %>">
+      <% if component.title.present? %>
+        <h2 class="margin-top-0">
+          <%= component.title %>
+        </h2>
+      <% end %>
+      <% if component.url.present? && component.url_link_text.present? %>
+        <a href="<%= component.url %>"
+            target="<%= get_link_target_attribute(component.url) %>"
+            title="Go to <%= component.url %>"
+            class="usa-link display-inline-block margin-top-3">
+          <%= component.url_link_text %>
+        </a>
+      <% end %>
+      <% if component.text.present? %>
+        <p><%= component.text.html_safe %></p>
+      <% end %>
+    </div>
+    <div class="<%= component.flipped_ratio? ? 'tablet:grid-col-4' : 'tablet:grid-col-8' %>">
+      <img class="usa-img" src="<%= component.image_s3_presigned_url %>" alt="<%= component.image_alt_text %>"/>
+    </div>
+  <% else %>
+    <div class="<%= component.flipped_ratio? ? 'tablet:grid-col-4' : 'tablet:grid-col-8' %>">
+      <img class="usa-img" src="<%= component.image_s3_presigned_url %>" alt="<%= component.image_alt_text %>"/>
+    </div>
+    <div class="<%= component.flipped_ratio? ? 'tablet:grid-col-8' : 'tablet:grid-col-4' %>">
+      <% if component.title.present? %>
+        <h2 class="margin-top-0">
+          <%= component.title %>
+        </h2>
+      <% end %>
+      <% if component.url.present? && component.url_link_text.present? %>
+        <a href="<%= component.url %>"
+            target="<%= get_link_target_attribute(component.url) %>"
+            title="Go to <%= component.url %>"
+            class="usa-link display-inline-block margin-top-3">
+          <%= component.url_link_text %>
+        </a>
+      <% end %>
+      <% if component.text.present? %>
+        <p><%= component.text.html_safe %></p>
+      <% end %>
+    </div>
+  <% end %>
```

Comment:
we can eliminate this half of the `<% if %>` block using the `order` utility class: 

e.g. if text_alignment is right, add an `order-first` class to the image div. This equivalent to setting an `order: -1` with plain CSS. https://designsystem.digital.gov/utilities/flex/#order